--- /usr/share/applications/org.speedcrunch.SpeedCrunch.desktop	2025-11-13 19:48:22.938833040 +0100
+++ ./org.speedcrunch.SpeedCrunch.desktop	2025-11-13 19:50:32.837767762 +0100
@@ -1,10 +1,12 @@
 [Desktop Entry]
 Name=SpeedCrunch
 Name[it]=SpeedCrunch
+GenericName=High precision calculator
+GenericName[it]=Calcolatrice di precisione
 Exec=speedcrunch
 Icon=org.speedcrunch.SpeedCrunch
 Type=Application
-Categories=Qt;Utility;Calculator;
+Categories=Mathematics;
 Keywords=arithmetic;calculation;math;scientific;
-Keywords[it]=aritmetica;calcolo;matematica;scientifico;
+Keywords[it]=aritmetica;calcolo;matematica;scientifico;scientifica;
 StartupWMClass=speedcrunch
