--- /usr/share/applications/kdiskmark.desktop	2025-06-24 23:56:26.736644042 +0200
+++ ./kdiskmark.desktop	2025-06-25 00:00:37.729352577 +0200
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Name=KDiskMark
-GenericName=KDiskMark
+GenericName=SSD/HDD Benchmark Tool
+GenericName[it]=Strumento di benchmark per SSD/HDD
 Exec=kdiskmark
 Icon=kdiskmark
 Terminal=false
 Type=Application
 Categories=System;
-Keywords=benchmark;storage;performance;speed;test;
-Keywords[it]=test;archiviazione;prestazioni;velocità;
-StartupNotify=false
+Keywords=disk;disks;storage;speed;performance;
+Keywords[it]=disco;dischi;storage;archiviazione;velocità;prestazioni;
+StartupNotify=true
