--- /usr/share/applications/kdiskmark.desktop	2024-09-07 18:28:18.476096430 +0200
+++ ./kdiskmark.desktop	2024-09-07 18:40:54.006956917 +0200
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Name=KDiskMark
-GenericName=KDiskMark
+GenericName=SSD/HDD Benchmark Tool
+GenericName[it]=Strumento di benchmark per SSD/HDD
 Exec=kdiskmark
 Icon=kdiskmark
 Terminal=false
 Type=Application
 Categories=System;
-Keywords=benchmark;disk;
-Keywords[it]=benchmark;disco;prestazioni;
-StartupNotify=false
+Keywords=disk;disks;speed;
+Keywords[it]=disco;dischi;velocità;prestazioni;
+StartupNotify=true
