--- /usr/share/applications/org.kde.plasma-systemmonitor.desktop	2025-04-28 20:14:33.814839387 +0200
+++ ./org.kde.plasma-systemmonitor.desktop	2025-04-28 20:15:57.802987471 +0200
@@ -1,12 +1,13 @@
 [Desktop Entry]
-Name=System Monitor
-Name[it]=Monitor di sistema
+Name=plasma-systemmonitor
+GenericName=System Monitor
+GenericName[it]=Monitor di sistema
 Exec=plasma-systemmonitor
 Icon=utilities-system-monitor
 Type=Application
 Categories=Qt;KDE;System;
-Keywords=task;manager;process;cpu;memory;
-Keywords[it]=processo;gestore;processo;cpu;memoria;
+Keywords=task;tasks;process;cpu;memory;resources;applications;daemons;crash;background;foreground;run;threads;
+Keywords[it]=processo;processi;cpu;memoria;risorse;applicazioni;daemon;demoni;crash;background;foreground;esecuzione;threads;
 SingleMainWindow=true
 X-DBUS-StartupType=Unique
 X-DBUS-ServiceName=org.kde.plasma-systemmonitor
