--- /usr/share/applications/org.gnome.Sysprof.desktop	2025-10-15 13:39:21.403115605 +0200
+++ ./org.gnome.Sysprof.desktop	2025-10-15 13:41:07.843763005 +0200
@@ -1,13 +1,12 @@
 [Desktop Entry]
 Name=Sysprof
-Name[it]=Sysprof
-GenericName=Profiler
-GenericName[it]=Profilatore
+GenericName=Performance Profiler
+GenericName[it]=Profiler di prestazioni
 TryExec=sysprof
 Exec=sysprof %u
 Icon=org.gnome.Sysprof
 Terminal=false
 Type=Application
 MimeType=application/x-sysprof-capture;
-Categories=GNOME;GTK;Development;Utility;
+Categories=System;
 StartupNotify=true
