--- /usr/share/applications/org.kde.drkonqi.coredump.gui.desktop	2024-03-17 14:30:37.016398713 +0100
+++ ./org.kde.drkonqi.coredump.gui.desktop	2024-03-17 19:55:01.604846210 +0100
@@ -1,6 +1,5 @@
 [Desktop Entry]
-Name=Crashed Processes Viewer
-Name[it]=Visore dei processi con arresto anomalo
+Name=drkonqi-coredump-gui
 GenericName=Crashed Processes Viewer
 GenericName[it]=Visore dei processi con arresto anomalo
 TryExec=coredumpctl
@@ -8,3 +7,5 @@
 Icon=tools-report-bug
 Type=Application
 Categories=Qt;KDE;System;
+Keywords=applications;
+Keywords=applicazioni;crash;interruzioni;
