--- /usr/share/applications/org.kde.drkonqi.coredump.gui.desktop	2025-09-19 16:26:02.173424192 +0200
+++ ./org.kde.drkonqi.coredump.gui.desktop	2025-09-19 16:28:18.410131713 +0200
@@ -1,6 +1,5 @@
 [Desktop Entry]
-Name=Crashed Processes Viewer
-Name[it]=Visore dei processi con arresto anomalo
+Name=drkonqi-coredump-gui
 GenericName=Crashed Processes Viewer
 GenericName[it]=Visore dei processi con arresto anomalo
 TryExec=coredumpctl
@@ -8,5 +7,5 @@
 Icon=tools-report-bug
 Type=Application
 Categories=Qt;KDE;System;
-Keywords=drkonqi;crash;coredumpd;bug;
-Keywords[it]=drkonqi;crash;coredumpd;bug;
+Keywords=drkonqi;crash;coredumpd;bug;applications;
+Keywords[it]=drkonqi;crash;coredumpd;bug;applicazioni;
