--- /usr/share/applications/zenmonitor.desktop	2024-03-17 14:30:37.029398588 +0100
+++ ./zenmonitor.desktop	2024-03-17 18:24:51.552201034 +0100
@@ -1,7 +1,11 @@
 [Desktop Entry]
-Name=Zenmonitor
+Name=Zenmonitor3
+GenericName=Monitor for AMD Ryzen CPUs
+GenericName[it]=Monitora CPU AMD Ryzen
 Exec=/usr/bin/zenmonitor
+Icon=/usr/share/pixmaps/zenmonitor.png
 Terminal=false
 Type=Application
-Categories=GTK;System;
-Keywords=CPU;AMD;zen;system;core;speed;clock;temperature;voltage;
+Categories=System;
+Keywords=system;processor;core;speed;frequency;clock;temperature;voltage;sensors;
+Keywords[it]=system;sistema;processore;core;speed;frequenza;clock;temperatura;tensione;sensori;
