--- /usr/share/applications/zenmonitor.desktop	2025-05-12 15:23:02.777943500 +0200
+++ ./zenmonitor.desktop	2025-05-12 15:25:48.202225757 +0200
@@ -1,8 +1,11 @@
 [Desktop Entry]
-Name=Zenmonitor
+Name=Zenmonitor3
+GenericName=Monitor for AMD Ryzen CPUs
+GenericName[it]=Monitora CPU AMD Ryzen
 Exec=/usr/bin/zenmonitor
-Icon=utilities-system-monitor
+Icon=/usr/share/pixmaps/zenmonitor.png
 Terminal=false
 Type=Application
-Categories=GTK;System;
-Keywords=CPU;AMD;zen;system;core;speed;clock;temperature;voltage;
+Categories=System;
+Keywords=system;processor;core;speed;frequency;clock;temperature;voltage;sensors;
+Keywords[it]=system;sistema;processore;core;speed;frequenza;clock;temperatura;tensione;sensori;
