--- /usr/share/applications/blender-4.2.desktop	2025-04-28 22:05:46.397155938 +0200
+++ ./blender-4.2.desktop	2025-04-28 22:14:07.102873043 +0200
@@ -1,13 +1,14 @@
 [Desktop Entry]
-Name=Blender Bin 4.2.4 4.2
-GenericName=3D modeler
-GenericName[it]=Modellatore 3D
-Exec=blender-4.2.4-4.2 %f
-Icon=blender-4.2.4-4.2
+Name=Blender
+GenericName=3D Modeling
+GenericName[it]=Modellazione 3D
+Exec=blender-4.2 %f
+Icon=blender
 Terminal=false
 Type=Application
 MimeType=application/x-blender;
-Categories=Graphics;3DGraphics;
-Keywords=3d;cg;modeling;animation;painting;sculpting;texturing;video editing;video tracking;rendering;render engine;cycles;python;
+Categories=Graphics;
+Keywords=animation;rendering;video;game engine;post-production;cad;
+Keywords[it]=animazione;rendering;video;game engine;post-produzione;cad;
 StartupWMClass=Blender
 PrefersNonDefaultGPU=true
