--- /usr/share/applications/com.github.johnfactotum.Foliate.desktop	2024-06-03 00:28:19.226781335 +0200
+++ ./com.github.johnfactotum.Foliate.desktop	2024-06-03 00:30:54.503123959 +0200
@@ -1,15 +1,15 @@
 [Desktop Entry]
 Name=Foliate
 Name[it]=Foliate
-GenericName=E-Book Viewer
-GenericName[it]=Visualizzatore di E-Book
+GenericName=Viewer for E-books
+GenericName[it]=Visore di e-book
 Exec=foliate %U
 Icon=com.github.johnfactotum.Foliate
 Terminal=false
 Type=Application
 MimeType=application/epub+zip;application/vnd.amazon.mobi8-ebook;application/vnd.comicbook+zip;application/x-fictionbook+xml;application/x-mobipocket-ebook;application/x-zip-compressed-fb2;x-scheme-handler/opds;
-Categories=Office;Viewer;
-Keywords=Ebook;Book;EPUB;Viewer;Reader;
-Keywords[it]=Ebook;Book;EPUB;Viewer;Reader;libri;visualizzatore;lettore;
+Categories=Graphics;
+Keywords=Ebook;Book;EPUB;Reader;
+Keywords[it]=Ebook;Book;EPUB;Lettore;Reader;libri;
 StartupNotify=true
 X-Purism-FormFactor=Workstation;Mobile;
