--- /usr/share/applications/gimp.desktop	2026-05-19 13:30:06.709498942 +0200
+++ ./gimp.desktop	2026-05-19 13:33:09.471900590 +0200
@@ -1,14 +1,15 @@
 [Desktop Entry]
-Name=GNU Image Manipulation Program
-Name[it]=GNU Image Manipulation Program
+Name=Gimp
 GenericName=Image Editor
+GenericName[it]=Editor di immagini
 TryExec=gimp-3.2
 Exec=gimp-3.2 %U
 Icon=gimp
 Terminal=false
 Type=Application
 MimeType=image/x-xcf;application/pdf;application/postscript;application/x-navi-animation;image/apng;image/avci;image/avif;image/bmp;image/dds;image/g3-fax;image/gif;image/heif;image/hej2k;image/jp2;image/jpeg;image/jxl;image/openraster;image/png;image/svg+xml;image/tiff;image/vnd.microsoft.icon;image/vnd.wap.wbmp;image/webp;image/x-dcm;image/x-dcx;image/x-exr;image/x-flic;image/x-icns;image/x-ico;image/x-jp2-codestream;image/x-pcx;image/x-pixmap;image/x-portable-anymap;image/x-psd;image/x-psp;image/x-seattle-filmcamera;image/x-sgi;image/x-sun-raster;image/x-tga;image/x-wmf;image/x-xbitmap;image/x-xwindowdump
-Categories=Graphics;2DGraphics;RasterGraphics;GTK;
-Keywords=GIMP;graphic;design;illustration;painting;
+Categories=Graphics;
+Keywords=images;photos;graphic;design;illustration;painting;
+Keywords[it]=immagine;foto;grafica;disegno;illustrazioni;pittura;fotoritocco;
 StartupWMClass=gimp
 StartupNotify=true
