--- /usr/share/applications/openscad.desktop	2025-05-08 20:44:04.505151102 +0200
+++ ./openscad.desktop	2025-05-08 20:51:48.902981878 +0200
@@ -1,8 +1,11 @@
 [Desktop Entry]
 Name=OpenSCAD
+GenericName=3D CAD Modelling
+GenericName[it]=Modellazione CAD 3D
 Exec=openscad %f
-Icon=openscad-nightly
+Icon=openscad
 Type=Application
 MimeType=application/x-openscad;
-Categories=Graphics;3DGraphics;Engineering;
-Keywords=3d;solid;geometry;csg;model;stl;
+Categories=Graphics;
+Keywords=solid;solids;stl;mechanics;models;geometry;csg;
+Keywords[it]=solido;solidi;stl;meccanica;modello;modelli;geometria;csg;
