--- /usr/share/applications/org.kde.spectacle.desktop	2025-12-26 23:40:24.734570422 +0100
+++ ./org.kde.spectacle.desktop	2025-12-26 23:45:35.101570520 +0100
@@ -6,9 +6,9 @@
 Exec=/usr/bin/spectacle
 Icon=spectacle
 Type=Application
-Categories=Qt;KDE;Utility;
-Keywords=snapshot;capture;print;screenshot;snipping;snipping tool;snip;
-Keywords[it]=schermata;cattura;stampa;screenshot;ritagliare;strumento di ritaglio;ritaglio;
+Categories=Qt;KDE;Graphics;
+Keywords=snapshot;capture;print;screenshot;snipping;snip;
+Keywords[it]=schermata;cattura;stampa;screenshot;ritaglia;ritaglio;
 StartupNotify=false
 DBusActivatable=true
 X-DBUS-StartupType=Unique
@@ -43,7 +43,7 @@
 
 [Desktop Action WindowUnderCursorScreenShot]
 Name=Capture Window Under Cursor
-Name[it]=Cattura finestra sotto il cursore
+Name[it]=Cattura la finestra sotto il cursore
 Exec=/usr/bin/spectacle -u
 X-KDE-Shortcuts=Meta+Ctrl+Print
 
@@ -67,5 +67,5 @@
 
 [Desktop Action OpenWithoutScreenshot]
 Name=Launch without taking a screenshot
-Name[it]=Esegui senza acquisire una schermata
+Name[it]=Avvia senza acquisire una schermata
 Exec=/usr/bin/spectacle -l
