--- /usr/share/applications/org.nomacs.ImageLounge.desktop	2025-01-01 14:14:25.153138398 +0100
+++ ./org.nomacs.ImageLounge.desktop	2025-01-01 14:38:55.625403969 +0100
@@ -1,9 +1,12 @@
 [Desktop Entry]
 Name=nomacs
 GenericName=Image Viewer
+GenericName[it]=Visore di immagini
 Exec=nomacs %f
 Icon=org.nomacs.ImageLounge
 Terminal=false
 Type=Application
 MimeType=image/avif;image/bmp;image/gif;image/heic;image/heif;image/jpeg;image/jxl;image/png;image/tiff;image/webp;image/x-eps;image/x-ico;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;
-Categories=Graphics;RasterGraphics;Viewer;2DGraphics;
+Categories=Graphics;
+Keywords=images;photos;
+Keywords[it]=immagine;foto;
