--- /usr/share/applications/uchmviewer.desktop	2025-04-06 23:13:10.664256970 +0200
+++ ./uchmviewer.desktop	2025-04-06 23:14:20.082917907 +0200
@@ -1,9 +1,12 @@
 [Desktop Entry]
 Name=uChmViewer
-GenericName=CHM file viewer
+GenericName=CHM File Viewer
+GenericName[it]=Visore di file CHM
 Exec=uchmviewer %f
 Icon=uchmviewer
 Terminal=false
 Type=Application
 MimeType=application/chm;application/x-chm;application/vnd.ms-htmlhelp;application/epub+zip
-Categories=Qt;KDE;Office;Viewer;
+Categories=Qt;KDE;Graphics;
+Keywords=ebooks;e-books;
+StartupNotify=true
