--- /usr/share/applications/ardour9.desktop	2026-06-29 11:58:41.211062470 +0200
+++ ./ardour9.desktop	2026-06-29 12:01:39.639872154 +0200
@@ -1,11 +1,15 @@
 [Desktop Entry]
-Name=Ardour 9
+Name=Ardour
+GenericName=Digital Audio Workstation
+GenericName[it]=Workstation audio digitale
 Exec=ardour9
 Icon=ardour9
 Terminal=false
 Type=Application
 MimeType=application/x-ardour;
-Categories=AudioVideo;Audio;X-AudioEditing;X-Recorders;X-Multitrack;X-Jack;
+Categories=AudioVideo;
+Keywords=microphone;instruments;track;record;capture;music;sounds;midi;
+Keywords[it]=microfono;strumenti;traccia;registrazione;cattura;musica;suoni;midi;
 StartupWMClass=Ardour
 X-NSM-Capable=true
 X-NSM-Exec=ardour9
