--- /usr/share/applications/audacity.desktop	2025-04-14 18:55:13.644574694 +0200
+++ ./audacity.desktop	2025-04-14 18:58:24.232033328 +0200
@@ -1,13 +1,14 @@
 [Desktop Entry]
 Name=Audacity
 GenericName=Sound Editor
-GenericName[it]=editore di suoni
+GenericName[it]=Editor audio
 Exec=env GDK_BACKEND=x11 UBUNTU_MENUPROXY=0 audacity %F
 Icon=audacity
 Terminal=false
 Type=Application
 MimeType=application/ogg;application/x-audacity-project;application/x-audacity-project+sqlite3;audio/aac;audio/ac3;audio/basic;audio/flac;audio/mp4;audio/x-aiff;audio/x-flac;audio/x-ms-wma;audio/x-vorbis+ogg;audio/x-wav;video/mpeg;
-Categories=AudioVideo;Audio;AudioVideoEditing;
-Keywords=sound;music editing;voice channel;frequency;modulation;audio trim;clipping;noise reduction;multi track audio editor;edit;mixing;WAV;AIFF;FLAC;MP2;MP3;
+Categories=AudioVideo;
+Keywords=audio;sounds;alsa;jack;editor;track;tracks;frequency;modulation;WAV;MP3;codec;
+Keywords[it]=audio;suono;suoni;alsa;jack;editor;traccia;tracce;frequenza;modulazione;WAV;MP3;codec;
 StartupWMClass=Audacity
-StartupNotify=false
+StartupNotify=true
