--- /usr/share/applications/com.rosegardenmusic.rosegarden.desktop	2025-01-08 04:42:01.869572093 +0100
+++ ./com.rosegardenmusic.rosegarden.desktop	2025-01-08 04:47:19.188331225 +0100
@@ -1,12 +1,14 @@
 [Desktop Entry]
 Name=Rosegarden
 GenericName=Sequencer and Notation Editor
+GenericName[it]=Sequencer ed editor musicale
 Exec=rosegarden %f
 Icon=rosegarden
 Terminal=false
 Type=Application
 MimeType=audio/midi;audio/x-rosegarden-composition;audio/x-rosegarden-device;audio/x-rosegarden-project;audio/x-rosegarden-template;
-Categories=AudioVideo;Audio;Midi;Sequencer;X-SuSE-Sequencer;X-Red-Hat-Base;
-Keywords=midi;audio;sequencer;music;notation;score;
+Categories=AudioVideo;
+Keywords=midi;audio;music;notation;score;
 StartupWMClass=Rosegarden
+StartupNotify=true
 X-KDE-NativeMimeType=audio/x-rosegarden-composition
