--- /usr/share/applications/fr.handbrake.ghb.desktop	2024-03-17 14:30:37.004398829 +0100
+++ ./fr.handbrake.ghb.desktop	2024-03-17 20:50:53.962694449 +0100
@@ -1,9 +1,12 @@
 [Desktop Entry]
 Name=HandBrake
-GenericName=Media Transcoder
+GenericName=Video Transcoder
+GenericName[it]=Convertitore video
 Exec=ghb %f
 Icon=fr.handbrake.ghb
 Terminal=false
 Type=Application
 MimeType=application/ogg;application/x-extension-mp4;application/x-flac;application/x-matroska;application/x-ogg;audio/ac3;audio/mp4;audio/mpeg;audio/ogg;audio/x-flac;audio/x-matroska;audio/x-mp2;audio/x-mp3;audio/x-mpeg;audio/x-vorbis;video/mp4;video/mp4v-es;video/mpeg;video/msvideo;video/ogg;video/quicktime;video/vnd.divx;video/x-avi;video/x-m4v;video/x-matroska;video/x-mpeg;video/x-ms-wmv;video/x-ogm+ogg;video/x-theora+ogg;x-content/video-dvd;x-content/video-svcd;x-content/video-vcd;
-Categories=GTK;AudioVideo;Video;
+Categories=AudioVideo;
+Keywords=CD;DVD;Bluray;transcoding;media;
+Keywords[it]=CD;DVD;Bluray;conversione;media;transcoding;
