--- /usr/share/applications/org.avidemux.Avidemux.desktop	2025-05-03 16:45:03.666573223 +0200
+++ ./org.avidemux.Avidemux.desktop	2025-05-03 16:49:17.911093645 +0200
@@ -1,9 +1,12 @@
 [Desktop Entry]
 Name=Avidemux
 GenericName=Video Editor
-Exec=avidemux3_qt5 %f
+GenericName[it]=Editor video
+Exec=avidemux3_qt5 --platform 'xcb' %f
 Icon=org.avidemux.Avidemux
 Terminal=false
 Type=Application
 MimeType=video/3gp;video/dv;video/flv;video/mkv;video/mp4;video/mpeg;video/quicktime;video/webm;video/x-anim;video/x-flv;video/x-m4v;video/x-matroska;video/x-mpeg;video/x-ms-asf;video/x-ms-wmv;video/x-msvideo;
-Categories=AudioVideo;AudioVideoEditing;Video;
+Categories=AudioVideo;
+Keywords=audio;tracks;transcoding;extraction;media;
+Keywords[it]=audio;tracce;conversione;estrazione;media;
