--- /usr/share/applications/org.avidemux.Avidemux.desktop	2025-11-11 03:57:38.294068029 +0100
+++ ./org.avidemux.Avidemux.desktop	2025-11-11 04:02:37.960330670 +0100
@@ -1,10 +1,13 @@
 [Desktop Entry]
 Name=Avidemux
 GenericName=Video Editor
+GenericName[it]=Editor video
 Exec=avidemux3_qt6 %f
 Icon=org.avidemux.Avidemux
 Terminal=false
 Type=Application
 MimeType=video/3gp;video/dv;video/flv;video/mkv;video/mp4;video/mpeg;video/quicktime;video/webm;video/x-anim;video/x-flv;video/x-m4v;video/x-matroska;video/x-mpeg;video/x-ms-asf;video/x-ms-wmv;video/x-msvideo;
-Categories=AudioVideo;AudioVideoEditing;Video;
+Categories=AudioVideo;
+Keywords=audio;tracks;transcoding;extraction;media;
+Keywords[it]=audio;tracce;conversione;estrazione;media;
 StartupWMClass=avidemux3_qt6
