--- /usr/share/applications/org.strawberrymusicplayer.strawberry.desktop	2024-03-17 14:30:37.044398443 +0100
+++ ./org.strawberrymusicplayer.strawberry.desktop	2024-03-17 20:49:16.805688856 +0100
@@ -1,34 +1,15 @@
 [Desktop Entry]
 Name=Strawberry
-GenericName=Strawberry Music Player
+GenericName=Music Player
+GenericName[it]=Lettore musicale
 TryExec=strawberry
 Exec=strawberry %U
 Icon=strawberry
 Terminal=false
 Type=Application
 MimeType=application/ogg;application/x-ogg;application/x-ogm-audio;audio/aac;audio/flac;audio/mp4;audio/mpeg;audio/mpegurl;audio/ogg;audio/vnd.rn-realaudio;audio/vorbis;audio/x-ape;audio/x-flac;audio/x-mp3;audio/x-mpeg;audio/x-mpegurl;audio/x-ms-wma;audio/x-musepack;audio/x-oggflac;audio/x-pn-realaudio;audio/x-scpls;audio/x-speex;audio/x-vorbis;audio/x-vorbis+ogg;audio/x-wav;audio/x-wavpack;video/x-ms-asf;x-content/audio-player;x-scheme-handler/tidal;
-Categories=AudioVideo;Player;Qt;Audio;
-Keywords=Audio;Player;
+Categories=AudioVideo;
+Keywords=audio;song;
+Keywords[it]=audio;canzoni;player;
 StartupWMClass=strawberry
 StartupNotify=false
-Actions=Play-Pause;Stop;StopAfterCurrent;Previous;Next;
-
-[Desktop Action Play-Pause]
-Name=Play/Pause
-Exec=strawberry --play-pause
-
-[Desktop Action Stop]
-Name=Stop
-Exec=strawberry --stop
-
-[Desktop Action StopAfterCurrent]
-Name=Stop after this track
-Exec=strawberry --stop-after-current
-
-[Desktop Action Previous]
-Name=Previous
-Exec=strawberry --previous
-
-[Desktop Action Next]
-Name=Next
-Exec=strawberry --next
