--- /usr/share/applications/org.pulseaudio.pavucontrol.desktop	2025-01-18 18:14:43.409511927 +0100
+++ ./org.pulseaudio.pavucontrol.desktop	2025-01-18 18:18:23.225550466 +0100
@@ -1,11 +1,11 @@
 [Desktop Entry]
-Name=Volume Control
-Name[it]=Regolazione del volume
-GenericName=Volume Control
-GenericName[it]=Regolazione del volume
+Name=Pavucontrol
+GenericName=PulseAudio Control
+GenericName[it]=Regolazioni di PulseAudio
 Exec=pavucontrol
 Icon=multimedia-volume-control
 Type=Application
-Categories=AudioVideo;Audio;Mixer;GTK;Settings;X-XFCE-SettingsDialog;X-XFCE-HardwareSettings;
+Categories=AudioVideo;
 Keywords=pavucontrol;PulseAudio;Microphone;Volume;Fade;Balance;Headset;Speakers;Headphones;Audio;Mixer;Output;Input;Devices;Playback;Recording;System Sounds;Sound Card;Settings;Preferences;
+Keywords[it]=pavucontrol;microfono;volume;dissolvenza;bilanciamento;auricolari;altoparlanti;cuffie;audio;mixer;uscita;ingresso;dispositivi;riproduzione;registrazione;suoni di sistema;scheda audio;configurazione;preferenze;
 StartupNotify=true
