--- a/app/compressfileitemaction.cpp	2025-03-06 00:02:17.016972299 +0100
+++ b/app/compressfileitemaction.cpp	2025-03-06 00:23:53.364565938 +0100
@@ -58,6 +58,8 @@
 
     compressMenu->addAction(createAction(icon, parentWidget, urlList, QStringLiteral("tar.gz")));
     actionsToBeDisabledInReadOnlyDir << compressMenu->actions().last();
+    compressMenu->addAction(createAction(icon, parentWidget, urlList, QStringLiteral("tar.bz2")));
+    actionsToBeDisabledInReadOnlyDir << compressMenu->actions().last();
 
     const QMimeType zipMime = QMimeDatabase().mimeTypeForName(QStringLiteral("application/zip"));
     // Don't offer zip compression if no zip plugin is available.
@@ -66,6 +68,13 @@
         actionsToBeDisabledInReadOnlyDir << compressMenu->actions().last();
     }
 
+    const QMimeType sevenZipMime = QMimeDatabase().mimeTypeForName(QStringLiteral("application/x-7z-compressed"));
+    // Don't offer 7z compression if no 7-zip plugin is available.
+    if (!m_pluginManager->preferredWritePluginsFor(sevenZipMime).isEmpty()) {
+        compressMenu->addAction(createAction(icon, parentWidget, urlList, QStringLiteral("7z")));
+        actionsToBeDisabledInReadOnlyDir << compressMenu->actions().last();
+    }
+
     compressMenu->addAction(createAction(icon, parentWidget, urlList, QString()));
 
     QAction *compressMenuAction = new QAction(i18nc("@action:inmenu Compress submenu in Dolphin context menu", "Compress"), parentWidget);
