--- /usr/share/applications/code.desktop	2025-11-26 21:06:39.487303924 +0100
+++ ./code.desktop	2025-11-26 21:09:49.899425572 +0100
@@ -1,18 +1,12 @@
 [Desktop Entry]
 Name=Visual Studio Code
-GenericName=Text Editor
+GenericName=Programming Editor
+GenericName[it]=Editor per programmazione
 Exec=/usr/bin/vscode --ozone-platform-hint=auto --enable-wayland-ime %F
 Icon=vscode
 Type=Application
 MimeType=application/x-code-workspace;
-Categories=TextEditor;Development;IDE;
+Categories=Development;
 Keywords=vscode;
 StartupWMClass=Code
-StartupNotify=false
-Actions=new-empty-window;
-
-[Desktop Action new-empty-window]
-Name=New Empty Window
-Name[it]=Nuova finestra vuota
-Exec=/usr/bin/vscode --ozone-platform-hint=auto --enable-wayland-ime --new-window %F
-Icon=vscode
+StartupNotify=true
