--- /usr/share/applications/code.desktop	2025-01-07 21:12:17.437115860 +0100
+++ ./code.desktop	2025-01-07 21:14:22.068897908 +0100
@@ -1,18 +1,12 @@
 [Desktop Entry]
-Name=VSCode
-GenericName=Code Editor
+Name=Visual Studio Code
+GenericName=Programming Editor
+GenericName[it]=Editor per programmazione
 Exec=/usr/bin/vscode --ozone-platform-hint=auto --enable-wayland-ime -- %F
 Icon=vscode
 Type=Application
 MimeType=application/x-code-workspace;inode/directory;text/plain;
-Categories=Utility;TextEditor;Development;IDE;
-Keywords=vscode;code;
+Categories=Development;
+Keywords=vscode;
 StartupWMClass=Code
-StartupNotify=false
-Actions=new-empty-window;
-
-[Desktop Action new-empty-window]
-Name=New Empty Window
-Name[it]=Nuova finestra vuota
-Exec=/usr/bin/vscode --ozone-platform-hint=auto --enable-wayland-ime --new-window
-Icon=vscode
+StartupNotify=true
