--- /usr/share/applications/designer-designer-5.desktop	2024-05-20 07:33:15.708429029 +0200
+++ ./designer-designer-5.desktop	2024-05-20 07:34:00.631008770 +0200
@@ -1,7 +1,10 @@
 [Desktop Entry]
-Name=Qt 5 Designer
+Name=Qt5 Designer
+GenericName=GUI Designer for Qt/KDE
+GenericName[it]=Progettazione GUI per QT/KDE
 TryExec=/usr/lib64/qt5/bin/designer
 Exec=/usr/lib64/qt5/bin/designer
 Icon=designer
 Type=Application
-Categories=Qt;Development;GUIDesigner;
+MimeType=application/x-designer;
+Categories=Qt;Development;
