--- /usr/share/applications/designer6-qttools-6.desktop	2024-05-20 07:28:40.634037244 +0200
+++ ./designer6-qttools-6.desktop	2024-05-20 07:31:15.450565349 +0200
@@ -1,7 +1,10 @@
 [Desktop Entry]
-Name=Qt 6 Designer
+Name=Qt6 Designer
+GenericName=GUI Designer for Qt/KDE
+GenericName[it]=Progettazione GUI per QT/KDE
 TryExec=designer6
 Exec=designer6
 Icon=designer6
 Type=Application
-Categories=Qt;Development;GUIDesigner;
+MimeType=application/x-designer;
+Categories=Qt;Development;
