--- /usr/share/applications/org.kde.kompare.desktop	2026-06-24 08:32:52.537473997 +0200
+++ ./org.kde.kompare.desktop	2026-06-24 08:34:35.016621401 +0200
@@ -1,14 +1,16 @@
 [Desktop Entry]
 Name=Kompare
 Name[it]=Kompare
-GenericName=Diff/Patch Frontend
-GenericName[it]=Interfaccia per diff e patch
+GenericName=Compare and Merge Tool
+GenericName[it]=Confronto e unione di file
 Exec=kompare %U
 Icon=kompare
 Terminal=false
 Type=Application
 MimeType=text/x-patch;
 Categories=Qt;KDE;Development;
-InitialPreference=10
+Keywords=diff;
+Keywords[it]=diff;merge;differenze;
+InitialPreference=95
 StartupWMClass=kompare
 X-DocPath=kompare/index.html
