--- /usr/share/applications/org.gnome.Meld.desktop	2024-06-05 00:43:44.655786029 +0200
+++ ./org.gnome.Meld.desktop	2024-06-05 00:46:05.159434138 +0200
@@ -1,14 +1,14 @@
 [Desktop Entry]
 Name=Meld
 Name[it]=Meld
-GenericName=Diff Viewer
-GenericName[it]=Visualizzatore di differenze
+GenericName=Compare and Merge Tool
+GenericName[it]=Confronto e unione di file
 Exec=meld %F
 Icon=org.gnome.Meld
 Terminal=false
 Type=Application
 MimeType=application/x-meld-comparison;
-Categories=GTK;Development;
-Keywords=diff;merge;
-Keywords[it]=diff;merge;differenze;unione;
+Categories=Development;
+Keywords=diff;
+Keywords[it]=diff;merge;differenze;
 StartupNotify=true
