--- /usr/share/applications/org.godotengine.Godot.desktop	2026-01-29 13:05:10.655780544 +0100
+++ ./org.godotengine.Godot.desktop	2026-01-29 13:07:18.573598269 +0100
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Name=Godot Engine
-GenericName=Libre game engine
+GenericName=Opensource 2D/3D Game Editor
+GenericName[it]=Editor di giochi 2D/3D opensource
 Exec=godot %f
 Icon=godot
 Terminal=false
 Type=Application
 MimeType=application/x-godot-project;
-Categories=Development;IDE;
-Keywords=game development;development;IDE;game engine;
+Categories=Development;
+Keywords=game;development;IDE;
 StartupWMClass=Godot
