--- /usr/share/applications/qdbusviewer.desktop	2024-05-20 07:35:01.858451704 +0200
+++ ./qdbusviewer.desktop	2024-05-20 07:35:59.755937050 +0200
@@ -1,7 +1,10 @@
 [Desktop Entry]
-Name=Qt 5 QDBusViewer
+Name=Qt5 QDBusViewer
+GenericName=Inspect D-Bus objects and messages
+GenericName[it]=Ispezione oggetti e messaggi D-Bus
 TryExec=/usr/lib64/qt5/bin/qdbusviewer
 Exec=/usr/lib64/qt5/bin/qdbusviewer
 Icon=qdbusviewer
 Type=Application
-Categories=Qt;Development;
+Categories=Development;
+StartupNotify=true
