--- a/3rdparty/backward-cpp/CMakeLists.txt	2025-07-15 14:39:08.000000000 +0200
+++ b/3rdparty/backward-cpp/CMakeLists.txt	2025-07-19 19:00:20.311007527 +0200
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 project(backward CXX)
 
 # Introduce variables:
--- a/3rdparty/backward-cpp/test_package/CMakeLists.txt	2025-07-15 14:39:08.000000000 +0200
+++ b/3rdparty/backward-cpp/test_package/CMakeLists.txt	2025-07-19 19:00:32.622967409 +0200
@@ -1,5 +1,5 @@
 project(backward-package-test)
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
