--- /usr/share/applications/firewall-config.desktop	2026-08-01 13:08:17.872579239 +0200
+++ ./firewall-config.desktop	2026-08-01 13:09:50.850021895 +0200
@@ -1,11 +1,13 @@
 [Desktop Entry]
-Name=Firewall
-Name[it]=Firewall
+Name=Firewall Configuration
+Name[it]=Configurazione del firewall
+GenericName=Firewall Configuration
+GenericName[it]=Configurazione del firewall
 Exec=firewall-config
 Icon=firewall-config
 Terminal=false
 Type=Application
-Categories=System;Settings;Security;
-Keywords=firewall;network;security;iptables;netfilter;
-Keywords[it]=firewall;rete;sicurezza;iptables;netfilter;
+Categories=Settings;
+Keywords=network;security;iptables;nftables;netfilter;fw;
+Keywords[it]=rete;sicurezza;iptables;nftables;netfilter;fw;
 StartupNotify=true
