--- /usr/share/applications/org.kde.kwalletmanager.desktop	2024-10-16 20:08:19.828863953 +0200
+++ ./org.kde.kwalletmanager.desktop	2024-10-16 20:17:55.140250179 +0200
@@ -1,13 +1,13 @@
 [Desktop Entry]
-Name=KWalletManager
-Name[it]=KWalletManager
+Name=KDE Wallet Management Tool
+Name[it]=Gestione del portafogli di KDE
 GenericName=KDE Wallet Management Tool
-GenericName[it]=Strumento per la gestione dei portafogli di KDE
+GenericName[it]=Gestione del portafogli di KDE
 Exec=kwalletmanager5 %F
 Icon=kwalletmanager
 Type=Application
 MimeType=application/x-kwallet;
-Categories=Qt;KDE;System;Security;
+Categories=Qt;KDE;Settings;
 InitialPreference=6
 DBusActivatable=true
 X-DocPath=kwallet5/index.html
