--- /usr/share/applications/libreoffice-draw.desktop	2024-06-30 05:53:09.225564566 +0200
+++ ./libreoffice-draw.desktop	2024-06-30 05:54:21.286936768 +0200
@@ -1,13 +1,15 @@
 [Desktop Entry]
 Name=LibreOffice Draw
-GenericName=Drawing Program
+GenericName=Vector Drawing
+GenericName[it]=Disegno vettoriale
 Exec=libreoffice --draw %U
 Icon=libreoffice-draw
 Terminal=false
 Type=Application
 MimeType=application/clarisworks;application/pdf;application/vnd.corel-draw;application/vnd.ms-publisher;application/vnd.oasis.opendocument.graphics;application/vnd.oasis.opendocument.graphics-flat-xml;application/vnd.oasis.opendocument.graphics-template;application/vnd.quark.quarkxpress;application/vnd.stardivision.draw;application/vnd.sun.xml.draw;application/vnd.sun.xml.draw.template;application/vnd.visio;application/x-pagemaker;application/x-stardraw;application/x-wpg;image/x-emf;image/x-freehand;image/x-wmf;
-Categories=Office;FlowChart;Graphics;2DGraphics;VectorGraphics;X-Red-Hat-Base;
-Keywords=Vector;Schema;Diagram;Layout;OpenDocument Graphics;Microsoft Publisher;Microsoft Visio;Corel Draw;cdr;odg;svg;pdf;vsd;
+Categories=Office;
+Keywords=Schema;Diagram;Layout;OpenDocument;Microsoft;Publisher;Visio;cdr;odg;svg;vsd;
+Keywords[it]=Schema;Diagramma;Layout;OpenDocument;Microsoft;Publisher;Visio;cdr;odg;svg;vsd;
 InitialPreference=5
 StartupWMClass=libreoffice-draw
 StartupNotify=true
@@ -17,5 +19,6 @@
 
 [Desktop Action NewDocument]
 Name=New Drawing
+Name[it]=Nuovo disegno
 Exec=libreoffice --draw
 Icon=document-new
