--- /usr/share/applications/libreoffice-base.desktop	2024-03-17 14:30:37.015398723 +0100
+++ ./libreoffice-base.desktop	2024-03-17 18:38:59.448527582 +0100
@@ -1,13 +1,15 @@
 [Desktop Entry]
 Name=LibreOffice Base
-GenericName=Database Development
+GenericName=Database Manager
+GenericName[it]=Gestore di database
 Exec=libreoffice --base %U
 Icon=libreoffice-base
 Terminal=false
 Type=Application
 MimeType=application/vnd.oasis.opendocument.base;application/vnd.sun.xml.base;
-Categories=Office;Database;X-Red-Hat-Base;
-Keywords=Data;SQL;
+Categories=Office;
+Keywords=data;queries;reports;SQL;OpenOffice;
+Keywords[it]=dati;interrogazioni;rapporti;SQL;OpenOffice;
 InitialPreference=5
 StartupWMClass=libreoffice-base
 StartupNotify=true
@@ -17,5 +19,6 @@
 
 [Desktop Action NewDocument]
 Name=New Database
+Name[it]=Nuovo database
 Exec=libreoffice --base
 Icon=document-new
