--- /usr/share/applications/org.kde.krename.desktop	2024-09-29 11:27:28.690462688 +0200
+++ ./org.kde.krename.desktop	2024-09-29 11:30:01.791162798 +0200
@@ -2,10 +2,12 @@
 Name=KRename
 Name[it]=KRename
 GenericName=Batch File Renamer
-GenericName[it]=Uno strumento di rinomina di massa
+GenericName[it]=Strumento di rinomina di massa
 Exec=krename -qwindowtitle "%c" %U
 Icon=krename
 Terminal=false
 Type=Application
-Categories=Qt;KDE;Utility;FileTools;
+Categories=Qt;KDE;Utility;
+Keywords=files;
+Keywords[it]=file;files;batch;rename;
 StartupWMClass=krename
