--- /usr/share/applications/org.kde.kleopatra.desktop	2025-03-05 23:53:16.804974288 +0100
+++ ./org.kde.kleopatra.desktop	2025-03-05 23:54:57.188407707 +0100
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Name=Kleopatra
 Name[it]=Kleopatra
-GenericName=Certificate manager and cryptography app
-GenericName[it]=Gestore di certificati e applicazione di crittografia
+GenericName=Certificate Manager and Crypto GUI
+GenericName[it]=Gestore di certificati e crittografia
 Exec=kleopatra
 Icon=kleopatra
 Type=Application
