--- /usr/share/applications/org.kde.plasma-welcome.desktop	2025-04-28 20:16:19.964762680 +0200
+++ ./org.kde.plasma-welcome.desktop	2025-04-28 20:18:04.323704427 +0200
@@ -1,11 +1,12 @@
 [Desktop Entry]
 OnlyShowIn=KDE;
-Name=Welcome Center
-Name[it]=Centro di benvenuto
+Name=plasma-welcome
+GenericName=Welcome Center
+GenericName[it]=Centro di benvenuto
 Exec=plasma-welcome
 Icon=start-here-kde-plasma
 Type=Application
 Categories=Qt;KDE;Documentation;
-Keywords=Welcome;Wizard;Onboarding;Contribute;Get involved;
-Keywords[it]=Benvenuto;Procedura guidata;Primo approccio;Contribuisci;Partecipa;
+Keywords=Wizard;Onboarding;Contribute;Get involved;
+Keywords[it]=Procedura guidata;Primo approccio;Contribuisci;Partecipa;
 SingleMainWindow=true
