--- /usr/share/applications/org.kde.plasma-welcome.desktop	2025-12-26 23:50:06.627094263 +0100
+++ ./org.kde.plasma-welcome.desktop	2025-12-26 23:51:25.497373352 +0100
@@ -1,5 +1,6 @@
 [Desktop Entry]
 OnlyShowIn=KDE;
+Name=plasma-welcome
 Name=Welcome Center
 Name[it]=Centro di benvenuto
 GenericName=Welcome Center
@@ -8,6 +9,6 @@
 Icon=start-here-kde-plasma
 Type=Application
 Categories=Qt;KDE;Documentation;
-Keywords=Welcome;Wizard;Onboarding;Contribute;Get involved;
-Keywords[it]=Benvenuto;Procedura guidata;Primo approccio;Contribuisci;Partecipa;
+Keywords=Wizard;Onboarding;Contribute;Get involved;
+Keywords[it]=Procedura guidata;Primo approccio;Contribuisci;Partecipa;
 SingleMainWindow=true
