--- /usr/share/applications/org.kde.krename.desktop	2024-03-17 14:30:37.055398337 +0100
+++ ./org.kde.krename.desktop	2024-03-17 17:55:00.023455490 +0100
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Name=KRename
 Name[it]=KRename
+GenericName=Batch renamer
+GenericName[it]=Strumento di rinomina di massa
 Exec=krename -qwindowtitle "%c" %U
 Icon=krename
 Terminal=false
 Type=Application
 Categories=Qt;KDE;Utility;
+Keywords=files;
+Keywords[it]=files;batch;rename;
