--- /usr/share/applications/opera.desktop	2025-09-19 16:23:25.952906255 +0200
+++ ./opera.desktop	2025-09-19 16:25:45.161585584 +0200
@@ -1,22 +1,23 @@
 [Desktop Entry]
 Name=Opera
-GenericName=Web browser
+GenericName=Web Browser
+GenericName[it]=Browser web
 TryExec=opera
 Exec=opera %U
 Icon=opera
 Terminal=false
 Type=Application
 MimeType=application/x-opera-download;application/xhtml_xml;text/html;text/xml;x-scheme-handler/ftp;x-scheme-handler/http;x-scheme-handler/https;
-Categories=Network;WebBrowser;
+Categories=Network;
 StartupWMClass=Opera
 Actions=new-window;new-private-window;
 
 [Desktop Action new-window]
 Name=New Window
+Name[it]=Nuova finestra
 Exec=opera --new-window
-X-TargetEnvironment=Unity
 
 [Desktop Action new-private-window]
 Name=New Private Window
+Name[it]=Nuova finestra anonima
 Exec=opera --incognito
-X-TargetEnvironment=Unity
