--- /usr/share/applications/org.telegram.desktop.desktop	2026-06-13 01:53:01.787616754 +0200
+++ ./org.telegram.desktop.desktop	2026-06-13 01:55:36.494819440 +0200
@@ -1,21 +1,23 @@
 [Desktop Entry]
 Name=Telegram
+GenericName=Instant Messaging
+GenericName[it]=Messaggistica istantanea
 TryExec=Telegram
 Exec=Telegram -- %U
-Icon=org.telegram.desktop
+Icon=telegram
 Terminal=false
 Type=Application
 MimeType=x-scheme-handler/tg;x-scheme-handler/tonsite;
-Categories=Chat;Network;InstantMessaging;Qt;
-Keywords=tg;chat;im;messaging;messenger;sms;tdesktop;
+Categories=Network;
+Keywords=chat;im;messaging;messenger;sms;telegram;tg;
+Keywords[it]=chat;im;messaggi;messenger;sms;telegram;tg;
 StartupWMClass=TelegramDesktop
 DBusActivatable=true
 SingleMainWindow=true
-X-GNOME-UsesNotifications=true
-X-GNOME-SingleWindow=true
 Actions=quit;
 
 [Desktop Action quit]
 Name=Quit Telegram
+Name[it]=Termina Telegram
 Exec=Telegram -quit
 Icon=application-exit
