--- /usr/share/applications/org.telegram.desktop.desktop	2025-06-13 15:23:48.111384222 +0200
+++ ./org.telegram.desktop.desktop	2025-06-13 16:09:59.193774722 +0200
@@ -1,21 +1,23 @@
 [Desktop Entry]
 Name=Telegram
+GenericName=Instant Messaging
+GenericName[it]=Messaggistica istantanea
 TryExec=/usr/bin/telegram-desktop
 Exec=/usr/bin/telegram-desktop -- %u
-Icon=org.telegram.desktop
+Icon=telegram
 Terminal=false
 Type=Application
 MimeType=x-scheme-handler/tg;x-scheme-handler/tonsite;
-Categories=Chat;Network;InstantMessaging;Qt;
-Keywords=tg;chat;im;messaging;messenger;sms;tdesktop;
+Categories=Network;
+Keywords=chat;im;messaging;messenger;sms;telegram;tg;
+Keywords[it]=chat;im;messaggi;messenger;sms;telegram;tg;
 StartupWMClass=TelegramDesktop
 DBusActivatable=true
 SingleMainWindow=true
-X-GNOME-UsesNotifications=true
-X-GNOME-SingleWindow=true
 Actions=quit;
 
 [Desktop Action quit]
 Name=Quit Telegram
+Name[it]=Termina Telegram
 Exec=/usr/bin/telegram-desktop -quit
 Icon=application-exit
