--- /usr/share/applications/org.kde.krdc.desktop	2024-03-17 14:30:37.062398270 +0100
+++ ./org.kde.krdc.desktop	2024-03-17 20:23:00.571740431 +0100
@@ -2,12 +2,12 @@
 Name=KRDC
 Name[it]=KRDC
 GenericName=Remote Desktop Client
-GenericName[it]=Client per connessione a desktop remoto
+GenericName[it]=Connessione a desktop remoto
 Exec=krdc -qwindowtitle %c %u
 Icon=krdc
 Terminal=false
 Type=Application
 MimeType=x-scheme-handler/vnc;x-scheme-handler/rdp
-Categories=Qt;KDE;Network;RemoteAccess;
+Categories=Network;
 StartupWMClass=krdc
 X-DocPath=krdc/index.html
