--- /usr/share/applications/org.telegram.desktop.desktop	2024-12-10 18:18:59.804581646 +0100
+++ ./org.telegram.desktop.desktop	2024-12-10 18:31:52.944663859 +0100
@@ -1,13 +1,16 @@
 [Desktop Entry]
 Name=Telegram
+GenericName=Instant Messaging
+GenericName[it]=Messaggistica istantanea
 TryExec=telegram-desktop
 Exec=telegram-desktop -- %u
 Icon=telegram
 Terminal=false
 Type=Application
 MimeType=x-scheme-handler/tg;x-scheme-handler/tonsite;
-Categories=Chat;Network;InstantMessaging;Qt;
-Keywords=tg;chat;im;messaging;messenger;sms;tdesktop;
+Categories=Network;
+Keywords=chat;im;messaging;messenger;sms;telegram;tg;
+Keywords[it]=chat;im;messaggi;messenger;sms;telegram;tg;
 StartupWMClass=TelegramDesktop
 DBusActivatable=true
 SingleMainWindow=true
@@ -17,5 +20,6 @@
 
 [Desktop Action quit]
 Name=Quit Telegram
+Name[it]=Termina Telegram
 Exec=telegram-desktop -quit
 Icon=application-exit
