--- /usr/share/applications/org.kde.krdc.desktop	2025-03-05 23:55:26.843271976 +0100
+++ ./org.kde.krdc.desktop	2025-03-05 23:56:54.889668024 +0100
@@ -2,12 +2,12 @@
 Name=KRDC
 Name[it]=KRDC
 GenericName=Remote Desktop Client
-GenericName[it]=Client per connessione a desktop remoto
+GenericName[it]=Connessione a desktop remoto
 Exec=krdc -qwindowtitle %c %u
 Icon=krdc
 Terminal=false
 Type=Application
 MimeType=application/x-krdc;x-scheme-handler/rdp;x-scheme-handler/vnc;
-Categories=Qt;KDE;Network;RemoteAccess;
+Categories=Network;
 StartupWMClass=krdc
 X-DocPath=krdc/index.html
