--- /usr/share/applications/org.kde.krdc.desktop	2026-03-02 05:09:13.441722800 +0100
+++ ./org.kde.krdc.desktop	2026-03-02 05:11:00.324768686 +0100
@@ -2,13 +2,13 @@
 Name=KRDC
 Name[it]=KRDC
 GenericName=Remote Desktop Client
-GenericName[it]=Client per connessione a desktop remoto
+GenericName[it]=Connessione a desktop remoto
 Exec=/usr/bin/krdc -qwindowtitle %c %u
 Icon=krdc
 Terminal=false
 Type=Application
 MimeType=application/x-krdc;x-scheme-handler/rdp;x-scheme-handler/vnc;
-Categories=Qt;KDE;Network;RemoteAccess;
+Categories=Network;
 StartupWMClass=krdc
 X-DocPath=krdc/index.html
 X-KDE-Wayland-Interfaces=org_kde_kwin_keystate
