--- /usr/share/applications/skypeforlinux.desktop	2024-03-17 14:30:37.124397672 +0100
+++ ./skypeforlinux.desktop	2024-03-17 20:12:08.355376675 +0100
@@ -1,17 +1,13 @@
 [Desktop Entry]
 Name=Skype Preview
+GenericName=Instant Messaging and VoIP
+GenericName[it]=Messaggistica istantanea e VoIP
 Exec=/usr/bin/skypeforlinux %U
 Icon=skypeforlinux
 Terminal=false
 Type=Application
 MimeType=x-scheme-handler/callto;x-scheme-handler/skype;x-scheme-handler/tel;
-Categories=Network;InstantMessaging;Telephony;
+Categories=Network;
 StartupWMClass=Skype
 StartupNotify=true
 X-KDE-Protocols=skype,tel,callto
-Actions=QuitSkype;
-
-[Desktop Action QuitSkype]
-Name=Quit Skype
-Name[it]=Esci da Skype
-Exec=/usr/bin/skypeforlinux --shutdown
