--- /usr/share/applications/thunderbird.desktop	2024-10-07 05:14:05.681447863 +0200
+++ ./thunderbird.desktop	2024-10-07 05:17:55.500659538 +0200
@@ -1,17 +1,20 @@
 [Desktop Entry]
-Name=Mozilla Thunderbird
-GenericName=Email
+Name=Thunderbird
+GenericName=Email Client
+GenericName[it]=Client email
 Exec=thunderbird %u
 Icon=thunderbird
 Terminal=false
 Type=Application
-MimeType=message/rfc822;text/calendar;x-scheme-handler/mailto;
-Categories=Network;Email;Calendar;
-Keywords=email;messages;calendar
+MimeType=application/x-xpinstall;message/rfc822;text/calendar;x-scheme-handler/mailto;x-scheme-handler/mid;x-scheme-handler/webcal;
+Categories=Network;
+Keywords=Email;E-mail;Newsgroup;Feed;RSS;messages;calendar
+Keywords[it]=Email;Mail;Posta;Newsgroup;Feed;RSS;messaggi;calendario
 StartupWMClass=thunderbird-esr
 StartupNotify=true
 Actions=profile-manager-window;
 
 [Desktop Action profile-manager-window]
-Name=Open the Profile Manager
+Name=Open Profile Manager
+Name[it]=Apri gestione profili
 Exec=thunderbird --ProfileManager
