--- /usr/share/applications/zoom.desktop	2024-11-22 21:36:14.348580340 +0100
+++ ./zoom.desktop	2024-11-22 21:37:23.554932697 +0100
@@ -1,8 +1,11 @@
 [Desktop Entry]
 Name=Zoom
+GenericName=Video Conference
+GenericName[it]=Video Conferenza
 TryExec=/opt/zoom/ZoomLauncher
 Exec=/opt/zoom/ZoomLauncher %U
 Icon=videoconference-zoom
 Type=Application
 MimeType=application/x-zoom;x-scheme-handler/zoommtg;x-scheme-handler/zoomus;
-Categories=Network;VideoConference;
+Categories=Network;
+X-KDE-Protocols=zoommtg;zoomus;tel;callto;zoomphonecall;
