--- /usr/share/applications/ZoomLauncher-zoom.desktop	2025-01-18 18:12:36.173645588 +0100
+++ ./ZoomLauncher-zoom.desktop	2025-01-18 18:14:00.960890139 +0100
@@ -1,8 +1,11 @@
 [Desktop Entry]
 Name=Zoom
+GenericName=Video Conference
+GenericName[it]=Video Conferenza
 TryExec=/opt/zoom/ZoomLauncher
 Exec=/opt/zoom/ZoomLauncher %U
 Icon=videoconference-zoom
 Type=Application
 MimeType=application/x-zoom;x-scheme-handler/zoommtg;x-scheme-handler/zoomus;
-Categories=Network;VideoConference;
+Categories=Network;
+X-KDE-Protocols=zoommtg;zoomus;tel;callto;zoomphonecall;
