--- /usr/share/applications/dosbox.desktop	2024-03-17 14:30:37.060398289 +0100
+++ ./dosbox.desktop	2024-03-17 19:05:35.417218276 +0100
@@ -1,7 +1,10 @@
 [Desktop Entry]
 Name=DOSBox
+GenericName=Emulator for DOS Games
+GenericName[it]=Emulatore per giochi DOS
 TryExec=dosbox
 Exec=dosbox
 Icon=/usr/share/pixmaps/dosbox.ico
 Type=Application
-Categories=Game;Emulator;
+Categories=Game;
+StartupNotify=false
