--- /usr/share/applications/org.wesnoth.Wesnoth.desktop	2024-03-17 14:30:37.107397836 +0100
+++ ./org.wesnoth.Wesnoth.desktop	2024-03-17 19:09:41.503701039 +0100
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Name=Battle for Wesnoth
-Name[it]=Battle for Wesnoth
+Name[it]=Battaglia di Wesnoth
 GenericName=Strategy Game
-GenericName[it]=Gioco strategico
+GenericName[it]=Gioco di strategia
 Exec=sh -c "wesnoth >/dev/null 2>&1"
 Icon=wesnoth-icon
 Type=Application
