--- /etc/xdg/menus/plasma-applications.menu	2025-04-02 05:38:24.000000000 +0200
+++ ./plasma-applications.menu	2025-04-28 20:55:07.079179553 +0200
@@ -335,10 +335,16 @@
 		</Menu>
 	</Menu>
 	<Menu>
+		<Name>Settingsmenu</Name>
+		<Directory>kf5-settingsmenu.directory</Directory>
+		<Include>
+			<Category>Settings</Category>
+		</Include>
+	</Menu>
+	<Menu>
 		<Name>System</Name>
 		<Directory>kf5-system.directory</Directory>
 		<Include>
-			<Category>Settings</Category>
 			<And>
 				<Category>System</Category>
 				<Not><Category>X-KDE-More</Category></Not>
