--- /usr/share/applications/org.stellarium.Stellarium.desktop	2026-03-06 03:45:55.753697202 +0100
+++ ./org.stellarium.Stellarium.desktop	2026-03-06 07:25:52.340451323 +0100
@@ -8,31 +8,26 @@
 Terminal=false
 Type=Application
 MimeType=application/x-stellarium-script;
-Categories=Qt;Education;Science;Astronomy;
-StartupNotify=false
-Actions=Window;Fullscreen;Debugmode;XCBPlatform;XCBPlatformTheme;
+Categories=Education;Science;
+StartupNotify=true
+Actions=Window;Fullscreen;XCBPlatform;XCBPlatformTheme;
 
 [Desktop Action Window]
 Name=Open in window
-Name[it]=Aprire in una finestra
+Name[it]=Apri in una finestra
 Exec=stellarium -f no
 
 [Desktop Action Fullscreen]
 Name=Open in fullscreen
-Name[it]=Aprire in modalità schermo intero
+Name[it]=Apri in modalità schermo intero
 Exec=stellarium -f yes
 
 [Desktop Action XCBPlatform]
 Name=Open with QPA platform XCB
-Name[it]=Aprire con QPA piattaforma XCB
+Name[it]=Apri con QPA piattaforma XCB
 Exec=stellarium -platform xcb
 
 [Desktop Action XCBPlatformTheme]
 Name=Open with QPA platform theme XCB
-Name[it]=Aprire con QPA piattaforma tema XCB
+Name[it]=Apri con QPA piattaforma tema XCB
 Exec=stellarium -platformtheme xcb
-
-[Desktop Action Debugmode]
-Name=Open in debug mode
-Name[it]=Aprire in modalità di debug
-Exec=stellarium --dump-opengl-details
